.github/workflows/e2e-tests.yml

```
@@ -0,0 +1,194 @@
+        with:
+          name: e2e-test-results
+          path: |
+            web/playwright-report/
+            web/test-results/
+            e2e-results.json
+          if-no-files-found: ignore
+
+      - name: Analyze E2E test results
+        id: analyze
+        if: always()
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2129:style:13:3: Consider using { cmd1; cmd2; } >> file instead of individual redirects

(shellcheck)